README.md

```diff
@@ -64,16 +58,4 @@ $ curl http://localhost:9410/api/v0/token -H 'content-type: application/json' -d
 }'
 ```
 
-## Configuration
```

Comment:
If we don't explain how to configure the system here, we need to explain it somewhere else.

Reply:
The fact that it's easy to do, doesn't mean people know it. If it's not here, it could indicate that either we didn't add it, or that there's nothing to be done. 

Keeping the docs up to date is just as important as keeping the tests running.